### DISCUSSION TITLE
Put profile picture of the user who added lang before its name #67

### DISCUSSION THREAD

**Discussion by amireshoon** [BlackIQ/Hello-World#68] [Ideas]
> Hello ,
> 
> We have a script that add creator and link after lang . But how about put the picture before !?
> 
> Example :
> 
> Assembly - Assembly - Added By  Taha Farahani
> Thanks .
> May the force be with you .
> Amir .

According to issue #67

**Comment by BlackIQ** [comment]
Hi .

I didn't know you made one too .
I'll delete that .

**Comment by amireshoon** [comment]
We can get contributor profile picture by this URL. `https://github.com/<username>.png?size=40`

**Reply by parsampsh** [reply]
I'll add this feature to `gen-readme.py`.
But we have a problem with Markdown. image is not inline block. is this possible to fix this with css in markdown?('display: inline')

**Reply by BlackIQ** [reply]
I use this :

<img src="https://avatars.githubusercontent.com/u/42321156?s=460&u=7003d9df79e8b8b1bf0f277308ba03a173491f94&v=4" width="20" height="20">

src = https://avatars.githubusercontent.com/u/42321156?s=460&u=7003d9df79e8b8b1bf0f277308ba03a173491f94&v=4
width = 20
height = 20

**Reply by parsampsh** [reply]
Thanks. i'll test it.

**Comment by parsampsh** [comment]
Please check #71

**Comment by amireshoon** [comment]
Pull request #72 Created a conflict in #71 I send a pull request to revert that.

**Reply by parsampsh** [reply]
i fixed conflict

**Reply by BlackIQ** [reply]
Tnx .
